playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml

```diff
@@ -127,37 +127,29 @@
     - name: Run image build and publish for cloud-provider-openstack
       shell:
         cmd: |
-          # NOTE(flaper87): Export these variables
-          # before setting -x and -e to avoid leaking
-          # sensitive data.
-          # NOTE(mrhillsman): Credentials still were leaking
-          # so removing the export unless absolutely needed
-          # and setting the login earlier instead of using Makefile
-          # https://github.com/kubernetes/cloud-provider-openstack/blob/master/Makefile#L255
-          #export DOCKER_USERNAME={{dockerhub.username}}
-          #export DOCKER_PASSWORD={{dockerhub.password}}
-          docker login -u {{dockerhub.username}} -p {{dockerhub.password}}
-          export REGISTRY=docker.io/k8scloudprovider
-          export VERSION=latest
-
-          set -x
-          set -e
-          set -o pipefail
-
-          if [ '{{ zuul.pipeline }}' != 'periodic' ]; then
-              exit 0;
+          if [ '{{ zuul.pipeline }}' == 'periodic' ] && [ '{{ zuul.branch }}' == 'master' ]; then
+            export LOG_DIR='{{ k8s_log_dir }}'
```

Comment:
Adding this because export does not make variables available across playbooks when executable is used; it creates a new shell session.